crates/ruma-federation-api/src/authenticated_media/get_content.rs

```diff
@@ -0,0 +1,106 @@
+//! `GET /_matrix/federation/*/media/download/{mediaId}`
+//!
+//! Retrieve content from the media store.
+
+pub mod unstable {
+    //! `/unstable/org.matrix.msc3916/` ([MSC])
+    //!
+    //! [MSC]: https://github.com/matrix-org/matrix-spec-proposals/pull/3916
```

Comment:
No, I forgot to say it in the description of the PR. It's not supposed to be mergeable right now because we need to wait for Matrix 1.11 to be out. But at least the logic for (de)serializing can be reviewed. I'll mark this as a draft to make sure it's not merged by accident.

Reply:
Stable endpoints added.